src/Xhgui/Controller/Run.php

```diff
@@ -9,6 +9,23 @@ public function __construct($app, $profiles, $watches)
         $this->_watches = $watches;
     }
 
+    public function homepage()
+    {
+        // TODO: how to fetch DIC from app?
+        global $di;
```

Comment:
I've wired all the controllers [into the di container](https://github.com/perftools/xhgui/blob/master/src/Xhgui/ServiceContainer.php#L116-L130). You can add the new dependencies there.